src/services/completions.ts

```diff
@@ -1068,12 +1068,12 @@ function getExhaustiveCaseSnippets(
         }
 
         const newClauses = map(elements, element => factory.createCaseClause(element, []));
-        const newLineChar = getNewLineCharacter(options, maybeBind(host, host.getNewLine));
+        const newLineChar = formatContext?.options.newLineCharacter ?? getNewLineOrDefaultFromHost(host);

```

Comment:
I'm confused; shouldn't these be:

```suggestion
        const newLineChar = getNewLineOrDefaultFromHost(host, formatContext?.options);
```

(That's what I get for replying earlier on my phone where I couldn't type it out)

Reply:
This was just me not realizing that was already the semantics of `getNewLineOrDefaultFromHost`